packages/ui/Marquee/MarqueeDecorator.js

```diff
@@ -377,7 +365,7 @@ const MarqueeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 				prevProps.forceDirection !== forceDirection
 			) {
 				this.cancelAnimation();
-			} else if (disabled && this.isHovered && marqueeOn === 'focus' && this.sync) {
+			} else if (this.isHovered && marqueeOn === 'focus' && this.sync) {
 				this.context.enter(this);
```

Comment:
We might have to keep `disabled` for this `MarqueeController` sync logic, or there might be some logic needed for `handleLeave` in `MarqueeController` .

With `disabled` remove from here, `enter` is registered into the controller, but `leave` is never called when blurred.